app/Services/Phoenix/Phoenix.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Rogue\Services\Phoenix;
+
+use DoSomething\Northstar\Common\RestApiClient;
+use DoSomething\Northstar\Resources\NorthstarClient;
+use DoSomething\Northstar\Resources\NorthstarUser;
+use DoSomething\Northstar\Resources\NorthstarUserCollection;
+use DoSomething\Northstar\Resources\NorthstarClientCollection;
+use GuzzleHttp\Cookie\CookieJar;
+use Cache;
+
+class Phoenix extends RestApiClient
+{
+	/**
+	* Create a new Phoenix API client.
+	* @param array $config
+	*/
+	public function __construct()
+	{
+        $this->base_uri = config('services.phoenix.uri') . '/api/' . config('services.phoenix.version') . '/';
+
+        $headers = [
+           'Content-type' => 'application/json',
+           'Accept' => 'application/json',
+        ];
+
+        parent::__construct($this->base_uri, $headers);
```

Comment:
It also doesn't look like `$this->base_uri` is a field on the base class, so you can make that into a normal variable `$url = config(...` before passing it to the parent constructor. It'll be stored on the RestApiClient's Guzzle instance.